### PR TITLE
Add Events to Lending Schema

### PR DESCRIPTION
**Pull request by melotik**

### CODE REVIEW

File: schemas/lending/SCHEMA.md [hunk 1]
```diff
@@ -71,4 +71,23 @@ Snapshot of the pool's state in the lending protocol.
...
+| log_index                | The event log. For transactions that don't emit event, create arbitrary index starting from 0. | number |
+| transaction_hash         | The hash of the transaction.                              | string |
+| transaction_signer       | The address that initiates the transaction (ie, the transaction signer). | string |
+| user_address             | The address that receives the output of the event (ie, account that receives aTokens in an Aave Deposit). | string |
+| pool_address             | The contract address of the pool.                         | string |
+| token_address            | The address of the underlying token that was interacted with (ie, USDC and not aUSDC in Aave). | string |
+| amount                   | The amount of token_address transacted, decimal normalized. | number |
+| amount_usd               | The amount of token_address transacted, in USD.           | number |
+| event_type               | The type of lending event, corresponds to the action taken by the user (ie, deposit, withdrawal, borrow, repay, liquidation, flashloan). | string |
```
Comment (0xroll) [inline schemas/lending/SCHEMA.md]: for liquidations, there are two tokens involved, collateral and debt amount, as well as liquidator and user that got liquidated.

not sure if having a separate schema (to include both tokens), or modifying this schema will be better?

Comment (melotik) [reply schemas/lending/SCHEMA.md]: Yeah, I thought about that, I wondered if we needed all that?

In this setup
- transaction_signer = the liquidator
- amount = the liquidated amount

Do we need the liquidatee and the debt amount/profit made?

Comment (melotik) [reply schemas/lending/SCHEMA.md]: If so, then I think we should have a new table for just liquidations

Comment (0xroll) [reply schemas/lending/SCHEMA.md]: good to have imo. better to have it now than adding it later in future

File: schemas/bridge/SCHEMA.md [hunk 2]
```diff
@@ -149,4 +149,20 @@ Snapshot of token level data.
...
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the claim.                               | number |
+| chain_id                 | The standard chain id.                                    | number |
+| transaction_hash         | The hash of the transaction.                              | string |
+| log_index                | The event log. For transactions that don't emit event, create arbitrary index starting from 0. | number |
+| user_address             | The address of the user who claimed the incentives.       | string |
+| claimed_token_address    | The smart contract address of the claimed token.          | string |
+| amount                   | The amount of the token claimed, decimal normalized.      | number |
+| amount_usd               | The amount of claimed tokens in USD.                      | number |
+| other_incentive_usd      | (Optional) Any incentives outside of the claimed token, in this transaction, summed up in USD terms. | number |
```
Comment (0xroll) [inline schemas/bridge/SCHEMA.md]: not sure if we want the tx_signer here. for scenarios where 1 EOA control many multisigs for claims

**Comment by 0xroll** [comment]
overall looks good to me, no other comments!

File: schemas/bridge/SCHEMA.md (hunk 2 above)
Comment (melotik) [reply schemas/bridge/SCHEMA.md]: Yep, I can add tx signer and claim address

**Review by bntyee — APPROVED** [review]

**Review by mac-obl — COMMENTED** [review]
lgtm!

**Pull request merged**